pkg/cmd/grafana-server/main.go

```diff
@@ -38,7 +38,7 @@ import (
 )
 
 // The following variables cannot be constants, since they can be overridden through the -X link flag
-var version = "5.0.0"
+var version = "7.5.0"
```

Comment:
Because I was running the backend directly without any wrappers, this was the default value for the `cfg.BuildVersion`, which is used to check compatibility against the gcom API. I can probably revert, but I didn't think there's any harm?

Reply:
I don't think there should be any harm :smile: 

But don't you see this when starting Grafana `INFO[05-12|10:34:18] Starting Grafana                         logger=server version=7.5.0-pre commit=4f03c7c9e6 branch=feat_httpclientprovider compiled=2021-05-11T20:14:22+0200     `? Or what do you mean without wrappers?

Reply:
Nah I'm special 😇 
`INFO[05-12|11:18:10] Starting Grafana logger=server version=5.0.0 commit=NA branch=main compiled=2021-05-12T11:18:10+0200`

It's because I build and run grafana-server directly, I don't use `make run` or anything so it doesn't go through the regular flow of setting the version from package.json 🙈

Reply:
I see. That's your own fault :smile: This version would potentially be out-of-date for you soon again so continuing updating this is not sustainable but would work for now.